src/app/series/directives/file-template.component.ts

```diff
@@ -1,33 +1,54 @@
 import { Component, Input } from '@angular/core';
-import { AudioFileTemplateModel } from '../../shared';
+import { AudioVersionTemplateModel, AudioFileTemplateModel } from '../../shared';
 
 @Component({
   selector: 'publish-file-template',
   styleUrls: ['file-template.component.css'],
   template: `
-    <div *ngIf="file && !file.isDestroy">
+    <div *ngIf="file && !file.isDestroy" class="file">
 
-      <publish-fancy-field textinput small required [model]="file" name="label" label="Segment {{file.position}} Label">
-        <span class="fancy-hint">A name for this audio segment, such as "Billboard" or "Part A"</span>
-      </publish-fancy-field>
-
-      <publish-fancy-field small class="length" [model]="file" label="Segment length in seconds" invalid="lengthAny">
-      <span class="fancy-hint">The minimum and maximum durations in seconds for this segment</span>
-
-        <publish-fancy-field number small inline hideinvalid [model]="file" name="lengthMinimum" label="Minimum">
+      <div class="label">
+        <publish-fancy-field textinput required hideinvalid [model]="file" name="label">
         </publish-fancy-field>
+      </div>
 
-        <publish-fancy-field number small inline hideinvalid [model]="file" name="lengthMaximum" label="Maximum">
-        </publish-fancy-field>
+      <div class="lengths">
+        <publish-fancy-duration [model]="file" tiny="true" name="lengthMinimum" label="Min"></publish-fancy-duration>
+        <publish-fancy-duration [model]="file" tiny="true" name="lengthMaximum" label="Max"></publish-fancy-duration>
+      </div>
 
```

Comment:
Hmm... yeah, i'll take a shot at making this at least somewhat responsive.  Probably only down to tablet-width, since Publish generally doesn't work so good when you get down to phone size.